osquery/tables/networking/windows/process_open_sockets.cpp

```diff
@@ -5,12 +5,36 @@
  *  This source code is licensed in accordance with the terms specified in
  *  the LICENSE file found in the root directory of this source tree.
  */
+#include <string>
+#include <vector>
 
 #include <osquery/logger.h>
 #include <osquery/tables.h>
 
 #include "win_sockets.h"
 
+namespace {
+const std::vector<std::string> winTcpStates = {
```

Comment:
This should be renamed to kWinTcpStates to match the coding style

Reply:
Got it. I'll make the change.